honeybee_energy/lib/schedules.py

```diff
@@ -0,0 +1,36 @@
+"""Establish the default schedule types within the honeybee_energy library."""
+from honeybee_energy.schedule.ruleset import ScheduleRuleset
+
+from ._loadschedules import _idf_schedules
+from .scheduletypelimits import activity_level
+
+
+# establish variables for the default schedules used across the library
+# and auto-generate materials if they were not loaded from default.idf
+try:
+    seated_activity = _idf_schedules['Seated Adult Activity']
+except KeyError:
```

Comment:
I definitely don't expect this to be the normal situation (the `except` here is truly exceptional). However, this code is here because we imagined allowing users to edit the default ,idf libraries, which now include the default constructions, materials, schedule types, and schedules.  So this code here is to catch the case that a user totally deletes this IDF schedule object that is critical for the rest of the library to run.  This way, the whole plugin doesn't crash if someone edits the IDF libraries incorrectly.

Reply:
Didn't we decide to separate the library files from the user one? similar to what we currently do in the legacy. For the exact same reasons that you mentioned I think the file that user edits should be separate from the file that we load as the library.

Reply:
Yes, there are two separate IDF files for `default.idf` and `user_library.idf`, which you can see here:
https://github.com/ladybug-tools/honeybee-energy/tree/master/honeybee_energy/lib/idf/constructions

And users could also add more idf files into that folder if they like and have them imported to the library (like if someone has a glazing construction that they exported from LBNL WINDOW, they can just drop it in that folder). In any case, having two separate files doesn't change the fact that the user can edit either depending on whether they want to change the defaults or just add new stuff on top of the defaults.  So my previous comment still holds and I am not sure how having separate files affects the fact that the we should still have a recovery system if the user edits the defaults incorrectly.